joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/parser/RubyJsonToNodeCreator.scala

```diff
@@ -672,8 +672,19 @@ class RubyJsonToNodeCreator(
     val body = obj
       .visitOption(ParserKeys.Body)
       .map {
-        case x: StatementList => x
-        case x                => StatementList(List(x))(x.span)
+        case x: StatementList =>
+          x.copy(statements = x.statements.map {
+            case _: AccessModifier =>
+              val ident = SimpleIdentifier()(x.span)
+              SimpleCall(ident, List.empty)(x.span)
+            case y => y
+          })(x.span)
+        case x: AccessModifier =>
```

Comment:
This may not be the best place to handle this, as there may be control structures too, etc. Could we not handle this in `AstCreator`? If the parent scope is method then make it a call instead of doing the usual access control.